packages/components/addon/components/hds/side-nav/portal/target.js

```diff
@@ -38,7 +39,7 @@ export default class SidenavPortalTarget extends Component {
 
   @action
   didUpdateSubnav(element, [count]) {
-    debounce(this, 'animateSubnav', element, [count], 100);
+    debounce(this, 'animateSubnav', element, [count], 0);
```

Comment:
can you add a comment on why we are leaving the `debounce`, but with a value of `0`?

Reply:
@didoo i left it out of caution incase we wanted to provide a way to configure that value. i updated my PR to just simply remove the debounce.